user_data_notebooks/oco2-mip-co2budget-yeargrid-v1_User_Notebook.ipynb

```diff
@@ -42,34 +42,12 @@
    "metadata": {},
```

Comment:
<p>The color scheme does not highlight the difference well, hard to find different pixels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/US-GHG-Center/ghgc-docs/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>yeah, we modified this one to match the color scheme on the webpage. It's hardcoded though which is not ideal.</p>